.claude/commands/test-design.md

```
@@ -0,0 +1,187 @@
+---
+allowed-tools: TodoWrite, TodoRead, Read, Write, MultiEdit, mcp__serena__find_file, mcp__serena__find_symbol, mcp__serena__list_memories, mcp__serena__search_for_pattern
+description: Create comprehensive test specification based on design document (Stage 3 of Specification-Driven Development)
+---
+
+## Context
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)